packages/app/package.json

```diff
@@ -14,50 +14,46 @@
     "lint": "backstage-cli package lint"
   },
   "dependencies": {
-    "@backstage/app-defaults": "1.5.3",
+    "@backstage/app-defaults": "1.5.4",
     "@backstage/catalog-model": "1.4.5",
-    "@backstage/cli": "0.26.2",
     "@backstage/config": "1.2.0",
-    "@backstage/core-app-api": "1.12.3",
-    "@backstage/core-components": "0.14.3",
-    "@backstage/core-plugin-api": "1.9.1",
-    "@backstage/integration-react": "1.1.25",
-    "@backstage/plugin-api-docs": "0.11.3",
-    "@backstage/plugin-catalog": "1.18.2",
+    "@backstage/core-app-api": "1.12.4",
+    "@backstage/core-components": "0.14.6",
+    "@backstage/core-plugin-api": "1.9.2",
+    "@backstage/integration-react": "1.1.26",
+    "@backstage/plugin-api-docs": "0.11.4",
+    "@backstage/plugin-catalog": "1.19.0",
     "@backstage/plugin-catalog-common": "1.0.22",
-    "@backstage/plugin-catalog-graph": "0.4.3",
-    "@backstage/plugin-catalog-import": "0.10.9",
-    "@backstage/plugin-catalog-react": "1.11.2",
-    "@backstage/plugin-home": "0.7.2",
-    "@backstage/plugin-org": "0.6.23",
-    "@backstage/plugin-permission-react": "0.4.21",
-    "@backstage/plugin-scaffolder": "1.19.2",
-    "@backstage/plugin-search": "1.4.9",
-    "@backstage/plugin-search-react": "1.7.9",
-    "@backstage/plugin-user-settings": "0.8.4",
-    "@backstage/theme": "0.5.2",
+    "@backstage/plugin-catalog-graph": "0.4.4",
+    "@backstage/plugin-catalog-import": "0.10.10",
+    "@backstage/plugin-home": "0.7.3",
+    "@backstage/plugin-org": "0.6.24",
+    "@backstage/plugin-permission-react": "0.4.22",
+    "@backstage/plugin-scaffolder": "1.19.3",
+    "@backstage/plugin-search": "1.4.10",
+    "@backstage/plugin-search-react": "1.7.10",
+    "@backstage/plugin-user-settings": "0.8.5",
+    "@backstage/theme": "0.5.3",
     "@emotion/react": "11.11.3",
     "@emotion/styled": "11.11.0",
     "@internal/plugin-dynamic-plugins-info": "*",
-    "@janus-idp/backstage-plugin-rbac-common": "1.4.0",
+    "@janus-idp/backstage-plugin-rbac-common": "1.4.2",
```

Comment:
do we need rbac-node and rbac-backend here too?

Reply:
This is the package for the frontend, so we won't need the backend dependencies here.

Reply:
yes